docs/scripting.md

```diff
@@ -4,45 +4,49 @@ permalink: /docs/scripting/
 
 # Scripting
 
-Hubot out of the box doesn't do too much but it is an extensible, scriptable robot friend. There are [hundreds of scripts written and maintained by the community](index.md#scripts) and it's easy to write your own.  You can create a custom script in hubot's `scripts` directory or [create a script package](#creating-a-script-package) for sharing with the community!
+Hubot out of the box doesn't do too much, but it is an extensible, scriptable robot friend. There are [hundreds of scripts written and maintained by the community](index.md#scripts) and it's easy to write your own. You can create a custom script in Hubot's `scripts` directory or [create a script package](#creating-a-script-package) for sharing with the community!
 
 ## Anatomy of a script
 
-When you created your hubot, the generator also created a `scripts` directory. If you peek around there, you will see some examples of scripts. For a script to be a script, it needs to:
+When you created your Hubot, the generator also created a `scripts` directory. If you peek around there, you will see some examples. For a script to be a script, it needs to:
 
-* live in a directory on the hubot script load path (`src/scripts` and `scripts` by default)
-* be a `.coffee` or `.js` file
-* export a function
+* live in a directory on the Hubot script load path (`src/scripts` and `scripts` by default)
+* be a `.js` file
+* export a function whos signature takes 1 argument (`robot`)
 
 By export a function, we just mean:
 
-```coffeescript
-module.exports = (robot) ->
-  # your code here
+```javascript
+module.exports = (robot) => {
+  // your code here
+}
 ```
 
-The `robot` parameter is an instance of your robot friend. At this point, we can start scripting up some awesomeness.
+The `robot` argument is an instance of your robot friend. At this point, we can start scripting up some awesomeness.
```

Comment:
I think technically Javascript refers to them as parameters, but they are effectively synonyms.